README.md

```diff
@@ -198,10 +198,18 @@ Please note that running all the test campaigns on the CloudLab Clemson c6420 co
 
 You can refer to [test_campaign.md](test_campaign.md) for detailed commands for running each test campaign.
 
-If you would like to try out an end-to-end test campaign, you can do it with the following command (taking the RabbitMQ operator as an example):
+If you would like to try out an end-to-end test campaign, you can do it with the following commands (taking the RabbitMQ operator as an example).
 
+Build the dependant modules as in previous sections if you haven't done so:
+
+```sh
+make
```

Comment:
Unsure 1/3: Should we mention it here?

Reply:
I think just ask people to run `make` should be fine just to be safe. If people forget to make, it would actually be very confusing to find out the error is due to missing module (the error message is buried in other log messages when running multiple workers)